docs/_includes/generated/jmx_server_configuration.html

```diff
@@ -1,18 +0,0 @@
-<table class="table table-bordered">
-    <thead>
-        <tr>
-            <th class="text-left" style="width: 20%">Key</th>
```

Comment:
this shouldn't be deleted; even if options are included in specific sections we also always generate a separate table for each originating file